test/parallel/test-benchmark-timers.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const common = require('../common');
+
+// Minimal test for timers benchmarks. This makes sure the benchmarks aren't
+// horribly broken but nothing more than that.
+
+const assert = require('assert');
+const fork = require('child_process').fork;
+const path = require('path');
+
+const runjs = path.join(__dirname, '..', '..', 'benchmark', 'run.js');
+const argv = ['--set', 'thousands=0.001',
```

Comment:
I kept the `thousands` and `millions` in the configuration names because that way it would be easier to read the benchmark results.